docs/macios/xcsync.md

```diff
@@ -1,15 +1,21 @@
 ---
-title: Xcode sync
-description: Learn how to use Xcsync, to provide .NET developers with improved support for editing Apple specific files.
+title: xcsync
```

Comment:
I think you also need to update the TOC files.
<img width="566" alt="image" src="https://github.com/user-attachments/assets/4827de3f-43ab-46d3-9472-7891f1043e87">

Reply:
I'd be inclined to leave the TOC title as is. Users should be able to glance at the TOC and have a clue what the doc is about.

Reply:
The issue is using term `Xcode`, which is a registered trademark of Apple, in a manner that can be mis-interpreted as using it in the name of our product or feature. "Syncing with Xcode" or something similar would be better, where it is obvious that the term is referring to the Apple product and not one of ours. I would prefer to use the same nomenclature that the rest of the .NET Tools do, but this page now is co-mingled with the VS Code MAUI extension documentation.

Reply:
updated to Syncing with Xcode

Reply:
Fine with me.